pvdeg/weather.py

```diff
@@ -11,19 +8,17 @@
 import glob
 import pandas as pd
 from rex import NSRDBX, Outputs
-from pvdeg import humidity
 import datetime
 import numpy as np
 import h5py
-import dask.dataframe as dd
 import xarray as xr
 
 
```

Comment:
```suggestion

     META_MAP = {
        "elevation": "altitude",
        "Elevation": "altitude",
        "Local Time Zone": "tz",
        "Time Zone": "tz",
        "timezone": "tz",
        "Dew Point": "dew_point",
        "Longitude": "longitude",
        "Latitude": "latitude",
    }
```
I can only suggest adding this, it won't let me suggest deleting the other lines

Reply:
Resolved, see #182